slices/libtbbmalloc2.yaml

```
@@ -0,0 +1,17 @@
+package: libtbbmalloc2
+
+essential:
+  - libtbbmalloc2_copyright
+
+slices:
+  libs:
+    essential:
+      - libc6_libs
+      - libgcc-s1_libs
+      - libstdc++6_libs
+    contents:
+      /usr/lib/*-linux-*/libtbbmalloc*.so.2*:
```

Nice, but let's please add those two shared objects on two different lines so that they are clearly visible upon slice inspection.

```suggestion
      /usr/lib/*-linux-*/libtbbmalloc.so.2*:
      /usr/lib/*-linux-*/libtbbmalloc_proxy.so.2*:
```